### PR TITLE
(test): adds test for `contact-details.component.tsx`

### PR DESCRIPTION
**Pull request by ayush-AI**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
In this PR, I have added tests for `contact-details.component.tsx`

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Comment by ayush-AI** [comment]
cc @Piumal1999 @anjula-sack @jayasanka-sack

File: packages/esm-patient-search-app/src/patient-search-page/patient-banner/contact-details/contact-details.test.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import { usePatientContactAttributes } from '../hooks/usePatientAttributes';
+
+const mockUseRelationships = useRelationships as jest.Mock;
+const mockUsePatientContactAttributes = usePatientContactAttributes as jest.Mock;
+
+const mockContactAttributes = [{ attributeType: { display: 'Email' }, value: 'test@example.com', uuid: 'email-uuid' }];
+
+const mockRelationships = [
+  {
+    uuid: 'relation-uuid',
```
Comment (anjula-sack) [inline packages/esm-patient-search-app/src/patient-search-page/patient-banner/contact-details/contact-details.test.tsx:14]: Shall we use an actual uuid? When it comes to mock data let's try to use data that similar to the actual values.

File: packages/esm-patient-search-app/src/patient-search-page/patient-banner/contact-details/contact-details.test.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  });
+  it('renders address', () => {
+    const mockAddress = {
+      postalCode: '12345',
+      address1: '123 Main St',
+      cityVillage: 'City',
+      stateProvince: 'State',
+      country: 'Country',
+      preferred: true,
+    };
```
Comment (anjula-sack) [inline packages/esm-patient-search-app/src/patient-search-page/patient-banner/contact-details/contact-details.test.tsx:48]: Shall we move this to mocks so we can reuse it?

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: packages/esm-patient-search-app/src/patient-search-page/patient-banner/contact-details/contact-details.test.tsx (hunk 1 above)
Comment (ayush-AI) [reply packages/esm-patient-search-app/src/patient-search-page/patient-banner/contact-details/contact-details.test.tsx]: done @anjula-sack

File: packages/esm-patient-search-app/src/patient-search-page/patient-banner/contact-details/contact-details.test.tsx (hunk 2 above)
Comment (ayush-AI) [reply packages/esm-patient-search-app/src/patient-search-page/patient-banner/contact-details/contact-details.test.tsx]: done @anjula-sack

**Review by anjula-sack — APPROVED** [review]
Thank you! @ayush-AI

**Pull request merged**